python-package/xgboost/plotting.py

```diff
@@ -14,7 +14,7 @@
 def plot_importance(booster, ax=None, height=0.2,
                     xlim=None, ylim=None, title='Feature importance',
                     xlabel='F score', ylabel='Features',
-                    importance_type='weight',
+                    importance_type='weight', max_features=None,
```

Comment:
this name is  a bit consuming. Can also be considered as maximum value of features, change to ```max_num_features``` instead?